docs/dapps/api-aliases.json

```diff
@@ -2,93 +2,125 @@
     "current": [
         {
             "from": "https://wallet.mainnet.alephium.org",
-            "to": "https://wallet-v15.mainnet.alephium.org"
+            "to": "https://wallet-v16.mainnet.alephium.org"
         },
         {
             "from": "https://wallet.testnet.alephium.org",
-            "to": "https://wallet-v15.testnet.alephium.org"
+            "to": "https://wallet-v16.testnet.alephium.org"
         },
         {
             "from": "https://backend.mainnet.alephium.org",
-            "to": "https://backend-v111.mainnet.alephium.org"
+            "to": "https://backend-v112.mainnet.alephium.org"
         },
         {
             "from": "https://backend.testnet.alephium.org",
-            "to": "https://backend-v111.testnet.alephium.org"
+            "to": "https://backend-v112.testnet.alephium.org"
         },
         {
             "from": "https://explorer.mainnet.alephium.org",
-            "to": "https://explorer-v111.mainnet.alephium.org"
+            "to": "https://explorer-v112.mainnet.alephium.org"
         },
         {
             "from": "https://explorer.testnet.alephium.org",
-            "to": "https://explorer-v111.testnet.alephium.org"
+            "to": "https://explorer-v112.testnet.alephium.org"
+        },
+        {
+            "from": "https://explorer.alephium.org",
+            "to": "https://explorer-v112.mainnet.alephium.org"
+        },
+        {
+            "from": "https://testnet.alephium.org",
+            "to": "https://explorer-v112.testnet.alephium.org"
         }
     ],
     "deprecated": [
+        {
+            "from": "https://wallet-v15.mainnet.alephium.org",
+            "to": "https://wallet-v16.mainnet.alephium.org"
+        },
+        {
+            "from": "https://wallet-v15.testnet.alephium.org",
+            "to": "https://wallet-v16.testnet.alephium.org"
+        },
         {
             "from": "https://wallet-v18.mainnet.alephium.org",
-            "to": "https://wallet-v15.mainnet.alephium.org"
+            "to": "https://wallet-v16.mainnet.alephium.org"
         },
         {
             "from": "https://wallet-v18.testnet.alephium.org",
-            "to": "https://wallet-v15.testnet.alephium.org"
+            "to": "https://wallet-v16.testnet.alephium.org"
```

Comment:
Thankfully there are no breaking changes between node 1.5.X and 1.6.X so this change will not cause a problem to users of the desktop wallet v1.4.0 (latest release).